weave

```diff
@@ -1972,20 +1981,10 @@ EOF
             shift
         done
         [ -n "$SUB_STATUS" ] || echo
-        if status=$(call_weave GET $STATUS_URL) ; then
-            case "$status" in
-                "")
-                    ;;
-                "404 page not found")
-                    echo "Invalid 'weave status' sub-command: $SUB_COMMAND" >&2
-                    usage
-                    ;;
-                *)
-                    echo "$status"
-                    ;;
-            esac
-        else
-            res=1
+        call_weave GET $STATUS_URL || res=$?
+        if [ $res = 404 ] ; then
```

Comment:
hah, i've just seen your "148 = 404 mod 256" trick above. `if [ $res -eq 148 ] ; ...` in the above does indeed work. But, as I pointed out, this relies on undefined behaviour.